### ISSUE TITLE
SelectedPath or SelectedField? 

### ISSUE THREAD

**Issue by Kilowhisky** [cibernox/ember-power-select#160]
I'm looking at this plugin as a replacement for the discontinued ember-select-2 and there is one big feature missing in this plugin. That is the ability to pass a set of objects to `options` and have the `selected` field accept a singular value which then looked up on the options for comparison. 

For example

// template.hbs
```
{{select-2
    content=languages
    optionValuePath='id'
    optionLabelPath='label'
    value=currentLanguage
}}
```

// component.js
```
export default Ember.Component.extend({

    /**
     * Translation service
     */
    i18n: Ember.inject.service(),

    /**
     * Defines all the available languages in the application
     */
    languages: function() {
        let i18n = this.get('i18n');
        return i18n.get('locales').map((item) => {
            return {
                id: item,
                label: i18n.t(`locales.${item}`)
            };
        });
    }.property('currentLanguage'),

    /**
     * Contains the currently selected language id
     * @property
     */
    currentLanguage: Ember.computed.alias('i18n.locale'),
});
```

Can this plugin do this or do i need to create a PR?

**Comment by cibernox** [comment]
I'd say that this is exactly what this component does...

**Comment by Kilowhisky** [comment]
Ok let me rephrase. 


In the documentation it says that the `selected` field compares by equality.

```
The component compares the selected option with the options in the collection by equality, to have in mind that when working with POJOs that { foo: 'bar' } === { foo: 'bar' } is false.
```

Is it possible to tell the component in the above example that the selected should compare the `foo` property instead of the whole object to determine selection?  I need to pass an array of objects to the `options` but i need the `selected` to work off of a property of the object.

**Comment by cibernox** [comment]
I think I get it now.

No, I don't think it would be good to add even another option for this.

I try to keep the code and the API as minimal as possible. Now if you pass an object that is not among the given options everything will work except that this element of the list won't be updated.

I'd suggest to find the translation among the options to have the object.
```
export default Ember.Component.extend({
    i18n: Ember.inject.service(),
    languages: function() {
        let i18n = this.get('i18n');
        return i18n.get('locales').map((item) => {
            return {
                id: item,
                label: i18n.t(`locales.${item}`)
            };
        });
    }.property('currentLanguage'),

    currentLanguage: Ember.computed('i18n.locale', function (){
      return this.get('languages').find(lang => lang.id === this.get('i18n.locale.id'));
    })
});
```

Does it make sense?

**Comment by Kilowhisky** [comment]
Except for the whole circular reference part of that...

Oh well looks like i won't be using this then. 

Thanks!
